config/daemon/systemd.md

```diff
@@ -151,6 +176,12 @@ you need to add this configuration in the Docker systemd service file.
     $ systemctl show --property=Environment docker
     Environment=HTTPS_PROXY=https://proxy.example.com:443/
     ```
+     
+    Note: for [rootless](https://docs.docker.com/engine/security/rootless/) docker, 
+    ```bash
+    $ systemctl --user show --property=Environment docker
+    ```
```

Comment:
I think it becomes quite difficult to read the instructions with a "note" for each step.

Perhaps we should create two separate tabs for "rootless" and "non-rootless" (similar to how we have tabs for different architectures on https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository

Reply:
This would certainly improve the readability.